docs/data-sources/dli_quotas.md

```diff
@@ -0,0 +1,55 @@
+---
+subcategory: "Data Lake Insight (DLI)"
+---
+
+# huaweicloud_dli_quotas
+
+Use this data source to get a list of quotas
+
+## Example Usage
+
+```hcl
+data "huaweicloud_dli_quotas" "test" {}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String) Specifies the region in which to query the resource.
+  If omitted, the provider-level region will be used.
+
+* `type` - (Optional, String) Specifies the type of quota.
+  Value options:
+   + **CU**: computing unit
```

Comment:
```suggestion
   + **CU**: Computing unit.
```
check the format and do not missing punctuation mark (.).

Reply:
done